### PR TITLE
Added revert reason

### PR DESCRIPTION
**Pull request by MadelineMurray**
Added revert reason

### CODE REVIEW

File: docs/Reference/Pantheon-CLI-Syntax.md [hunk 1]
```diff
@@ -1000,6 +1000,31 @@ privacy-url="http://127.0.0.1:8888"
...
+```bash tab="Configuration File"
+revert-reason-enabled=true
+```
+
+Enables including the [revert reason](../Using-Pantheon/Transactions/Revert-Reason.md) in the transaction 
+receipt. Default is `false`. 
+
+!!! caution 
+    Enabling revert reason may use a significant amount of memory. We do not recommend enabling revert
+    reason when connected to the Ethereum mainnet.
```
Comment (rain-on) [inline docs/Reference/Pantheon-CLI-Syntax.md]: "public Ethereum networks", not just mainnet.

File: docs/Using-Pantheon/Transactions/Revert-Reason.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
+description: Including revert reason in transactions 
+<!--- END of page meta data -->
+
+# Revert Reason 
+
+In contracts, the [`revert`](https://solidity.readthedocs.io/en/v0.5.10/control-structures.html#revert) function
```
Comment (rain-on) [inline docs/Using-Pantheon/Transactions/Revert-Reason.md]:
```suggestion
In smart contracts, the [`revert`](https://solidity.readthedocs.io/en/v0.5.10/control-structures.html#revert) operation 
```

File: docs/Using-Pantheon/Transactions/Revert-Reason.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
+description: Including revert reason in transactions 
+<!--- END of page meta data -->
+
+# Revert Reason 
+
+In contracts, the [`revert`](https://solidity.readthedocs.io/en/v0.5.10/control-structures.html#revert) function 
+triggers an exception to flag an error and revert the current call. An optional string message containing 
+information about the error is passed back to the caller.
```
Comment (rain-on) [inline docs/Using-Pantheon/Transactions/Revert-Reason.md]: returned to the client from the EVM (not to the caller per se).

File: docs/Using-Pantheon/Transactions/Revert-Reason.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+    ```
+
+## Enabling Revert Reason 
+
+Use the [`--revert-reason-enabled`](../../Reference/Pantheon-CLI-Syntax.md#revert-reason-enabled) command line option
+to include the revert reason in the transaction receipt in Pantheon. 
+
+!!! caution 
+    Enabling revert reason may use a significant amount of memory. We do not recommend enabling revert
+    reason when connected to the Ethereum mainnet.
```
Comment (rain-on) [inline docs/Using-Pantheon/Transactions/Revert-Reason.md]: same text as elsewhere? (I put a comment against it)

File: docs/Using-Pantheon/Transactions/Revert-Reason.md [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+Use the [`--revert-reason-enabled`](../../Reference/Pantheon-CLI-Syntax.md#revert-reason-enabled) command line option
+to include the revert reason in the transaction receipt in Pantheon. 
+
+!!! caution 
+    Enabling revert reason may use a significant amount of memory. We do not recommend enabling revert
+    reason when connected to the Ethereum mainnet. 
+
+## Where is the Revert Reason Included 
+
+When revert reason is enabled, the revert reason is included in the transaction receipt and returned by
```
Comment (rain-on) [inline docs/Using-Pantheon/Transactions/Revert-Reason.md]:
```suggestion
When revert reason is enabled, the revert reason is included in the transaction receipt returned by 
```

File: docs/Using-Pantheon/Transactions/Revert-Reason.md [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+         "transactionIndex": "0x0",
+         "revertReason":"Not enough Ether provided"
+       }
+    }
+    ```
+    
+## Dapp support 
+
+Client libraries (eg, web3j) do not support extracting the revert reason from the transaction receipt. 
+To extract the revert reason your Dapp must interact directly with Pantheon using a custom JSON -> Object
```
Comment (rain-on) [inline docs/Using-Pantheon/Transactions/Revert-Reason.md]: do people like "dApp" or "Dapp"?

File: docs/Reference/Pantheon-CLI-Syntax.md (hunk 1 above)
Comment (MadelineMurray) [reply docs/Reference/Pantheon-CLI-Syntax.md]: Fixed

File: docs/Using-Pantheon/Transactions/Revert-Reason.md (hunk 2 above)
Comment (MadelineMurray) [reply docs/Using-Pantheon/Transactions/Revert-Reason.md]: Fixed

File: docs/Using-Pantheon/Transactions/Revert-Reason.md (hunk 3 above)
Comment (MadelineMurray) [reply docs/Using-Pantheon/Transactions/Revert-Reason.md]: Fixed

File: docs/Using-Pantheon/Transactions/Revert-Reason.md (hunk 4 above)
Comment (MadelineMurray) [reply docs/Using-Pantheon/Transactions/Revert-Reason.md]: Fixed

File: docs/Using-Pantheon/Transactions/Revert-Reason.md (hunk 5 above)
Comment (MadelineMurray) [reply docs/Using-Pantheon/Transactions/Revert-Reason.md]: Fixed

File: docs/Using-Pantheon/Transactions/Revert-Reason.md (hunk 6 above)
Comment (MadelineMurray) [reply docs/Using-Pantheon/Transactions/Revert-Reason.md]: It's dapp or Dapp per the Consensys editorial guidelines

**Pull request merged**